assets/css/default.css

```diff
@@ -41,3 +41,7 @@ footer .icons a {
   display: inline-block;
   margin: 0 .25em;
 }
+
+.title {
+  padding: 20px;
```

Comment:
Why we need this? Could you give the screenshot?

I think we need to setup a CI here for preview.

Reply:
![screenshot from 2018-05-11 23-29-01](https://user-images.githubusercontent.com/24368528/39935463-4355c72e-5573-11e8-931f-a0951f4a5c0c.png)

Reply:
The main problem is the `clip-path` property. However, we could accept it now and we need to fix it ASAP.